boilerplate/app/app.tsx.ejs

```diff
@@ -1,83 +1,55 @@
-// Welcome to the main entry point of the app.
-//
-// In this file, we'll be kicking off our app or storybook.
-
+/**
+ * Welcome to the main entry point of the app. In this file, we'll
+ * be kicking off our app or storybook.
+ *
+ * Most of this file is boilerplate and you shouldn't need to modify
+ * it very often. But take some time to look through and understand
+ * what is going on here.
+ *
+ * The app navigation resides in ./app/navigation, so head over there
+ * if you're interested in adding screens and navigators.
+ */
```

Comment:
I like this intro. If we're going to be a boilerplate, lets help people do things!